src/webgl/3d_primitives.js

```diff
@@ -1575,7 +1575,7 @@ p5.RendererGL.prototype.quad = function(x1, y1, z1, x2, y2, z2, x3, y3, z3, x4,
     }
     quadGeom
       .computeNormals()
-      ._makeTriangleEdges()
+      ._makeQuadEdges()
```

Comment:
Nice work on the implementation of this! I wonder if maybe instead of making it a method on `p5.Geometry`, we could just inline it here, since it only can be used for single-quad shapes like this? So that would be something like:
```js
quadGeom.computeNormals();

const vertexOrder = [0, 2, 3, 1];
for (let i = 0; i < vertexOrder.length; i++) { /* etc */ }

quadGeom.edgesToVertices();
```

Reply:
Ok @davepagurek doing the changes.